lib/curl_sha512_256.c

```diff
@@ -269,9 +268,6 @@ Curl_sha512_256_finish(unsigned char *digest,
 
 #else /* No system or TLS backend SHA-512/256 implementation available */
 
-/* Use local implementation */
-#define HAS_SHA512_256_IMPLEMENTATION   1
```

Comment:
I'm not sure about the intent here, the macro is set twice after the last use.
Hopefully the code is OK.